src/dotnet/Chat.UI.Blazor/Components/PlaceSettings/PlaceSettingsEditTypeModalPage.razor

```diff
@@ -0,0 +1,221 @@
+                <InviteList
+                    Invites="m.Invites"
+                    LinkFormat="@InviteLinkFormat.PrivateChat"
+                    NewInviteId="@_newInviteId"
+                    AllowRevoke="true"/>
+            }
+            <Button IsDisabled="@(m.Invites.Count >= MaxInvitesCount)" Click="OnNewInviteClick" Class="btn-modal h-10 btn-w-full">
+                <Icon><i class="icon-plus text-xl"></i></Icon>
+                <Title>New private join link</Title>
+            </Button>
+        </FormBlock>
+    }
+</Form>
+
+
+@code {
+    private const int MaxInvitesCount = 5;
+    private Form?_formRef;
+    private FormModel? _form;
+    private Symbol _newInviteId = Symbol.Empty;
+    private DialogButtonInfo _submitButtonInfo = null!;
+
+    [Inject] private ChatUIHub Hub { get; init; } = null!;
+    [Inject] private ComponentIdGenerator ComponentIdGenerator { get; init; } = null!;
+    [Inject] private DiffEngine DiffEngine { get; init; } = null!;
+
+    private Session Session => Hub.Session();
+    private IPlaces Places => Hub.Places;
+
+    [CascadingParameter] public DiveInModalPageContext Context { get; set; } = null!;
+
+    private PlaceId PlaceId { get; set; }
+
+    protected override void OnInitialized() {
+        PlaceId = Context.GetTypedModel<PlaceId>();
+        Context.SetTitle("Edit place settings");
+        _submitButtonInfo = DialogButtonInfo.CreateSubmitButton("Save", OnSubmit);
+        Context.RegisterButtons(DialogButtonInfo.CancelButton, _submitButtonInfo);
+        base.OnInitialized();
+    }
+
+    protected override async Task OnInitializedAsync() {
+        var place = await Places.Get(Session, PlaceId, default).Require();
+        _form = new FormModel(ComponentIdGenerator) {
+            IsPublic = place.IsPublic,
+        };
+        Context.SetClass("edit-chat-type");
+        await base.OnInitializedAsync();
+    }
+
+    protected override ComputedState<ComputedModel>.Options GetStateOptions()
+        => ComputedStateComponent.GetStateOptions(GetType(),
+            static t => new ComputedState<ComputedModel>.Options() {
+                InitialValue = ComputedModel.Loading,
+                Category = ComputedStateComponent.GetStateCategory(t),
+            });
+
+    protected override async Task<ComputedModel> ComputeState(CancellationToken cancellationToken) {
+        var placeId = PlaceId;
+
+        var place = await Places.Get(Session, placeId, default).Require();
+        if (!place.Rules.CanEditProperties())
+            return new() { Place = place };
+
+        var invites = await Hub.Invites.ListPlaceInvites(Session, placeId, cancellationToken);
+        var threshold = Hub.Clocks().SystemClock.Now - TimeSpan.FromDays(3);
+        var activeInvites = invites
+            .Where(c => c.ExpiresOn > threshold)
+            .OrderByDescending(c => c.ExpiresOn)
+            .ToList();
+        var link = Links.Chat(placeId.ToRootChatId()).ToDisplayUrl(Hub.UrlMapper());
```

Comment:
We should use a link to place welcome chat. Link to place root chat is for internal usage only.

Reply:
See https://github.com/Actual-Chat/actual-chat/blob/a6034b0bab4cd87f3cff73908133d42b2880401b/src/dotnet/Chat.UI.Blazor/Components/PlaceSettings/PlaceSettingsStartModalPage.razor#L217

Reply:
Done